.github/workflows/release.yml

```diff
@@ -49,3 +53,16 @@ jobs:
       release-branch: 'release-proxy'
     secrets:
       ci-access-token: ${{ secrets.CI_ACCESS_TOKEN }}
+
+  create-compute-release-branch:
+    if: inputs.create-compute-release-branch
+
+    permissions:
+      contents: write
+
+    uses: ./.github/workflows/_create-release-pr.yml
+    with:
+      component-name: 'Compute'
```

Comment:
Could you please also change `component-name: 'Storage & Compute'` above to just `component-name: 'Storage'`

Reply:
Good catch, thanks